TemplePlus/dispatcher.cpp

```diff
@@ -72,6 +77,40 @@ int DispatcherReplacements::Dispatch54AoE(objHndl handle, DispIO* evtObj, D20Dis
 	return 0;
 }
 
+int DispatcherReplacements::GetNumNaturalAttacks_(objHndl handle) {
+	if (!handle) return 0;
+	auto obj = objSystem->GetObject(handle);
```

Comment:
For temple+ please also check obj.
Although rare, we sometimes get invalid non-null handles, which return a null object.